src/js/control-bar/time-controls/time-display.js

```diff
@@ -86,7 +87,13 @@ class TimeDisplay extends Component {
         return;
       }
 
-      const oldNode = this.textNode_;
+      let oldNode = this.textNode_;
+
+      if (oldNode && !this.contentEl_.contains(oldNode)) {
+        oldNode = null;
+
+        log.warn('Prevented to execute \'replaceChild\' on \'Node\' since the node to be replaced was not a child of this node.');
```

Comment:
perhaps this should instead be: "Prevented TimeDisplay#updateTextnode_ from running as the content el no longer contains the text node element."

Reply:
That was a bit clearer, but it's not prevented from running, it's just changing its behavior. What do you say about the following?

```suggestion
        log.warn('TimeDisplay#updateTextnode_: Prevented to replace text node element since it was no longer a child of this node. Appending a new node instead.');
```

Reply:
"Prevented replacement of" in that case

Reply:
```suggestion
        log.warn('TimeDisplay#updateTextnode_: Prevented replacement of text node element since it was no longer a child of this node. Appending a new node instead.');
```